report/sections/process-perspective/scaling-strategy.tex

```diff
@@ -0,0 +1,4 @@
+\subsection{Strategy for scaling and load balancing}
+We have deployed all of our components with Kubernetes deployments\cite{docs:kubernetes-deployment}, and configured them to run behind Kubernetes services\cite{docs:kubernetes-service}, this ensures component level load balancing.
+Using deployments also allows us to manually scale the number of instances of each component. While deployments do have the ability to autoscale, after some configuration, we have decided against it due to cost issues and because our monitoring indicated that the system was not getting enough load to justify scaling.\\
+Deployments also support zero downtime updates. This is achieved by changing settings on the deployment, such as the image to be used, then Kubernetes creates a new pod, and then kills an old pod when the new is ready, repeating the process until all pods have the new configuration.\\
```

Comment:
The document is set up so that the standard LaTeX "indented" paragraphs are replaced by normal paragraphs with space between:
```suggestion
Using deployments also allows us to manually scale the number of instances of each component. While deployments do have the ability to autoscale, after some configuration, we have decided against it due to cost issues and because our monitoring indicated that the system was not getting enough load to justify scaling.

Deployments also support zero downtime updates. This is achieved by changing settings on the deployment, such as the image to be used, then Kubernetes creates a new pod, and then kills an old pod when the new is ready, repeating the process until all pods have the new configuration.
```